integration/access_log_json_test.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"bufio"
```

Comment:
Sometimes the formatter does this automatically. I don't understand how to cause it to do so.